content/en/docs/components/pipelines/concepts/metadata.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "ML Metadata"
+description = "Conceptual overview about Metadata in Kubeflow Pipelines"
+weight = 90
+                    
++++
+
+**Note:** Kubeflow Pipelines has been migrated from [kubeflow/metadata](https://github.com/kubeflow/metadata)
+to [google/ml-metadata](https://github.com/google/ml-metadata) for Metadata dependency.
+
+Kubeflow Pipelines output information of *Executions*, *Artifact* of a pipeline run,
```

Comment:
Is there a grammar issue here? The relationship between Executions and Artifacts and the other information listed is not clear (at least to me) as written.

Reply:
I have rewritten so that the `runtime information about a pipeline run` stands out, and a link to MLMD official doc to explain Executions/Artifacts, etc.